### PR TITLE
1195/mk/use remote packages

### LINKED ISSUE #1195: package versioning

### PR DESCRIPTION
**Pull request by mkurapov**
## Changes proposed in this pull request

- Replacing (removing) `open-payments` `openapi` `http-signature-utils` in favour of npm published libraries:
    - Updating all of the imports 
    - Updating Github Actions pipelines
    - See: https://github.com/interledger/open-payments/pull/245

~Once this PR has been merged, all packages will be bumped to "1.0.0" and re-tested:~
- https://github.com/interledger/open-payments/pull/246 

## Context

- Fixes #1195 

## Checklist

- [x] Related issues linked using `fixes #number`
- [ ] Tests added/updated
- [ ] Documentation added
- [ ] Make sure that all checks pass

### CODE REVIEW

File: packages/auth/package.json [hunk 1]
```diff
@@ -36,13 +36,13 @@
     "node-mocks-http": "^1.12.2",
     "objection": "^3.0.1",
     "objection-db-errors": "^1.1.2",
-    "open-payments": "workspace:*",
-    "openapi": "workspace:../openapi",
+    "@interledger/open-payments": "1.0.0-alpha.1",
```
Comment (wilsonianb) [inline packages/auth/package.json:39]:
```suggestion
    "@interledger/open-payments": "1.0.0",
```
?

**Review by wilsonianb — COMMENTED** [review]
I'm jealous of your ~10k removed lines

File: packages/auth/package.json (hunk 1 above)
Comment (sabineschaller) [reply packages/auth/package.json:39]: We did `alpha` for now and will bump as soon as we are sure it all plays well together

**Review by sabineschaller — DISMISSED** [review]
Checked it out
- sanity ✅
- localenv build and start ✅
- postman demo ✅ 

Let's do it!

**Comment by sabineschaller** [comment]
Wait, just saw that one of the checks isn't properly configured:

> Code scanning results / CodeQL Completed in 3s — 1 configuration not found

File: packages/auth/package.json (hunk 1 above)
Comment (mkurapov) [reply packages/auth/package.json]: @sabineschaller I tested with the alpha versions previously in the PR, after publishing 1.0.0, we can use 1.0.0 for all of the packages, ie yes, that's correct @wilsonianb, I've just pushed the change

**Review by wilsonianb — APPROVED** [review]
>Let's do it!

**Pull request merged**